.travis.yml

```
@@ -7,7 +7,26 @@ python:
 before_install:
   - python3 -m pip install --upgrade pip setuptools wheel
 install:
-  - pip3 install git+https://github.com/nextstrain/cli
+  # https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/use-conda-with-travis-ci.html#the-travis-yml-file
+  - if [[ "$TRAVIS_PYTHON_VERSION" == "2.7" ]]; then
+      wget https://repo.continuum.io/miniconda/Miniconda2-latest-Linux-x86_64.sh -O miniconda.sh;
+    else
+      wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh;
+    fi
```

Since we have:

https://github.com/nextstrain/zika/blob/b2aab9f7e06514277763bc17a28320f624607379/.travis.yml#L5-L6

It should be safe to just get the Miniconda3 installer:

```suggestion
  - wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh;
```